docs/03_tutorials/20_tic-tac-toe-game-smart-contract-single node.md

```diff
@@ -0,0 +1,761 @@
+2. Create key pair - [How To Create Key Pairs](https://developers.eos.io/manuals/eos/latest/cleos/how-to-guides/how-to-create-key-pairs)
+3. Create the account - [How To Create An Account](https://developers.eos.io/manuals/eos/latest/cleos/how-to-guides/how-to-create-an-account#gatsby-focus-wrapper)
+4. Import the account private key to the wallet - [How To Import A Key](https://developers.eos.io/manuals/eos/latest/cleos/how-to-guides/how-to-import-a-key)
+
+For this tutorial we need to create two accounts
+1. The host - the host will load the smart contract
+2. The challenger - the challenger will play the game with the host.
+
+#### Commands
+To use the command line to create the accounts run the following commands. Make sure that the local wallet is [open](https://developers.eos.io/manuals/eos/latest/cleos/command-reference/wallet/open) and [unlocked](https://developers.eos.io/manuals/eos/latest/cleos/command-reference/wallet/unlock) and run these commands to create the accounts with the private keys stored in the local wallet.
+
+Create a wallet called 'local'.
+
+```shell
+cleos wallet create --name local --to-console
+```
+
+This command outputs to stdout
+```console
+Creating wallet: local
+Save password to use in the future to unlock this wallet.
+Without password imported keys will not be retrievable.
+"PW5K8dtLxB8q56JbT8UA5FQyDykNT24LS6i6gcBuxbYSixFgEpFa7"
+```
+
+Save the password, you will need this to unlock the wallet later.
+
+To check the wallet run
+```shell 
+cleos wallet list
+```
+
+The output shows two wallets, one for each version of this tutorial, and * shows that the local wallet is unlocked.
+```console
+Wallets:
+[
+  "local *",
+  "testnet"
+]
+```
+Create two key pairs, one for each account.
+
+```shell
+cleos create key --to-console
+```
+```console
+Private key: 5JSRUrUVbRsV2yJ2XSMtRtPzQ5UKbSYEGEjdKfGMS1xsvRZj7FH
+Public key: EOS5p55prHwrN6KosqF4NdRayVW2mqwA8RGNEbaZXRBs2SQHwBWSf
+```
+```shell
+cleos create key --to-console
+```
+```console
+Private key: 5JReVMTiiztAUyQGp9w7BMMm1HVUurDmEKuSL53DQww3JKVZjot
+Public key: EOS7qkiVnptc8wbHzHPC9jj1YECKJgQeUktBTm8RDA64oH3e75QW5
+```
+
+Create the host account.  
+```shell
+cleos create account eosio host EOS5p55prHwrN6KosqF4NdRayVW2mqwA8RGNEbaZXRBs2SQHwBWSf
+```
+
+Import the matching private key to the local wallet
+```shell
+cleos wallet import --name local --private-key 5JSRUrUVbRsV2yJ2XSMtRtPzQ5UKbSYEGEjdKfGMS1xsvRZj7FH
+```
+
+Create the challenger account and import a private key into the local wallet. 
+```shell
+cleos create account eosio challenger EOS7qkiVnptc8wbHzHPC9jj1YECKJgQeUktBTm8RDA64oH3e75QW5
+```
+
+Import the matching private key to the local wallet
+```shell
+cleos wallet import --name local --private-key 5JReVMTiiztAUyQGp9w7BMMm1HVUurDmEKuSL53DQww3JKVZjot
+```
+
+[[note | Note]]
+| You can also use three or more accounts, one for the smart contract and separate accounts for the host(s) and the challenger(s).
+
+[[warning | Keep your keys safe]]
```

Comment:
Are you sure you want to show the callout in red color? If you want to display it in yellow color (more like a warning), you can use 'caution' instead: "[[caution | Keep your keys safe]]" - I know Thorka kind of got this backwards (warning should be danger, and caution should be warning).